Dockerfile

```diff
@@ -2,7 +2,13 @@ FROM 056154071827.dkr.ecr.us-east-1.amazonaws.com/base-image-ruby-version-arg:2.
 MAINTAINER cru.org <wmd@cru.org>
 
 ARG RAILS_ENV=production
+ARG DD_API_KEY
+RUN DD_INSTALL_ONLY=true DD_API_KEY=$DD_API_KEY bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
```

Comment:
I'm not really keen on how this references 'master', since it means we don't have repeatable builds. But it's probably ok.

Reply:
Agreed, but those are the instructions they gave us:  https://app.datadoghq.com/account/settings#agent/ubuntu

Reply:
Just learned that this installs agent version 6 by default, but version 7 is out now. It requires `DD_AGENT_MAJOR_VERSION=7 ` (https://docs.datadoghq.com/agent/versions/upgrade_to_agent_v7/?tab=linux)

We should add that at some point. That will at least give us major-version repeatability.

Reply:
Oh, gotcha.  I'll give myself a task to do that on all the rails apps at some point.